src/subapps/projects/components/ResourcesPane.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { Collapse } from 'antd';
+import ResizeObserver from 'resize-observer-polyfill';
+
+import ActionButton from './ActionButton';
+
+const { Panel } = Collapse;
+
+import './ResourcesPane.less';
+
+const ResourcesPane: React.FC<{}> = ({ children }) => {
+  const [paneWidth, setPaneWidth] = React.useState<number>();
+
+  React.useEffect(() => {
+    const parentDiv = document.getElementsByClassName('activity-view')[0];
```

Comment:
get the parent div by using a React.useRef and query ref.current.parent instead of asking for the class name. this way, we will always get the parent div even if we change the class name or something.